webapp-django/crashstats/api/jinja2/api/documentation.html

```diff
@@ -105,17 +105,17 @@ <h2><a href="#{{ endpoint.name }}">{{ endpoint.name }}</a></h2>
             {% endfor %}
           </tbody>
         </table>
-      {% else %}
-        <p>
-          <em>Takes no parameters</em>
-        </p>
```

Comment:
I decided this isn't wildly helpful and it's erroneous for API endpoints that have complicated payloads.